### ISSUE TITLE
Update Documentation to reflect current state of the program

### ISSUE THREAD

**Issue by hollunder** [dalboris/vpaint#77]
Hi there,

the documentation is still at 1.5 and possibly does not even cover that completely.
It would be very helpful if it would cover 1.6 including the beta features.

As it stands I can't even figure out how to make animations outside of motion paste. I would like to interpolate between two existing frames and assume that one of the other animation options or the animation cycle editor allows me to do that, but I have no idea how.

**Comment by dalboris** [comment]
Indeed, the documentation doesn't cover a lot, I'm really sorry about that! I'll see if I can do some effort on this side to improve it, especially the animation cycle editor which I have to agree is unusable without documentation (and still quite hard to use even knowing how it works...). Though, keep in mind that at the moment, the tools are quite limited anyway on what can be done, so do not keep your expectations too high ;) I expect it will take 2 more years before VPaint is really "usable".

**Comment by hollunder** [comment]
I'm not an artist and for the simple thing I have currently planned vpaint will do. The question is how easily.

I tried some more and found that the beta feature 'inbetween selection' does nothing more often than not. Could you explain what I can do or must not do for this feature to work?

**Comment by hollunder** [comment]
Same thing for the 'inbetween face', which I could not get to work at all.
I tried selecting two faces on two frames, which seemed like the most logical thing to do, but no cigar.

**Comment by dalboris** [comment]
Unfortunately, as you have discovered, 'inbetween selection' is not yet implemented for faces. It only works if you select one path at a given frame, and another path at another frame. By 'path', I mean either one edge, or a sequence of connected edges.

Currently, there are only two ways to create an inbetween face that interpolates two existing key faces:
1. Write it by hand in the file
2. Using the animated cycle editor, which is basically the same as 1., just a bit more convenient.

In both cases, it requires understanding of the underlying data structure, which is complicated, and if you don't do it right VPaint may crash. I'll try to give some basic documentation on how to do 2. but it's far from intuitive. In most cases, you're better of selecting the face at frame 1, motion paste it at frame 2, then 'glue' it's edges to existing edges at frame 2.

It's a bit clunky, I totally agree: it's simply a temporary workaround to allow me doing research. The intended workflow is to use 'inbetween selection', like your intuition told you, but it's unfortunately not yet implemented for faces, since it's quite complicated to implement (in fact, it's an open problem, but making it work in simple cases is doable).

**Comment by vpaintanimation** [comment]
In fact, I also have been looking at all of Mr Boris, I look forward to Mr Boris complete vpaint core rewrite, but I look forward to vpaint intermediate frames to choose for the function of the face, it must be a very complex, because the face may contain many vertices and the number of edges, and faces in the middle of the object frame interpolation has many complex situations, the realization of the function of this will make vpaint a major step to improve. A lot of people can not help, I can only wish you had more time and less stress,

**Comment by dalboris** [comment]
One thing to not forget to add is the existence of the autosave feature: #85
